### PR TITLE
Add french i18n for login and email

### LINKED ISSUE #11949: Update French translations
### Describe the bug

Logout and idp login pages are missing French translations

### Version

18.0.0

### Expected behavior

Logout and IDP login option text translated in Frrench

### Actual behavior

Logout and idp login pages completely or partially in English when selected locale is French

### How to Reproduce?

Add French locale to realm supported locales

For the login IDP text
- Add an IDP to the realm
- Go to the realm login page
- Select French locale
- Text "Or sign in with" below the login/password fields is in English

For the logout page
- Login to a administration realm
- Sign out
- The logout confirmation page is completely in English

### Anything else?

PR incoming

### PR DESCRIPTION
**Pull request by Crow-EH**
Closes #11949

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties [hunk 1]
```diff
@@ -183,6 +255,7 @@ federatedIdentityEmailExistsMessage=Cet utilisateur avec ce courriel existe d\u0
 confirmLinkIdpTitle=Ce compte existe d\u00e9j\u00e0
 federatedIdentityConfirmLinkMessage=L''utilisateur {0} {1} existe d\u00e9j\u00e0. Que souhaitez-vous faire ?
 federatedIdentityConfirmReauthenticateMessage=Identifiez vous afin de lier votre compte avec {0}
+nestedFirstBrokerFlowMessage=The {0} user {1} is not linked to any known user.
```
Comment (FranckKe) [inline themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:258]:
```suggestion
nestedFirstBrokerFlowMessage=L''utilisateur {0} {1} n'est li\u00e9 \u00e0 aucun utilisateur connu.```

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties [hunk 2]
```diff
@@ -79,29 +90,62 @@ region=\u00c9tat, Province ou R\u00e9gion
...
 oauthGrantRequest=Voulez-vous accorder ces privil\u00e8ges d''acc\u00e8s ?
 inResource=dans
 
+oauth2DeviceVerificationTitle=Connexion de l''appareil
+verifyOAuth2DeviceUserCode=Entrez le code \u00e0 usage unique fourni par votre appareil et cliquez sur Soumettre.
+oauth2DeviceInvalidUserCodeMessage=Code invalide, veuillez r\u00e9\u00e9ssayer.
+oauth2DeviceExpiredUserCodeMessage=Le code a expir\u00e9. Veuillez r\u00e9\u00e9ssayer de vous connecter depuis votre appareil.
+oauth2DeviceVerificationCompleteHeader=Connexion de l''appareil r\u00e9ussie
+oauth2DeviceVerificationCompleteMessage=Vous pouvez fermer cette fen\u00eatre de navigateur et retourner sur votre appareil.
+oauth2DeviceVerificationFailedHeader=Device Login Failed
```
Comment (FranckKe) [inline themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties:144]:
```suggestion
oauth2DeviceVerificationFailedHeader=\u00c9chec de la connexion au dispositif
```

**Review by FranckKe — COMMENTED** [review]
Overral all good except a few things:
I think `adresse courriel` is more correct than `adresse de courriel`  https://fr.wikipedia.org/wiki/Adresse_%C3%A9lectronique.
And `rééssayer` => `réessayer`

**Comment by Crow-EH** [comment]
Thank you for your review @FranckKe

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties [hunk 3]
```diff
@@ -79,29 +90,62 @@ region=\u00c9tat, Province ou R\u00e9gion
...
 country=Pays
 emailVerified=Courriel v\u00e9rifi\u00e9
+website=Page web
+phoneNumber=Num\u00e9ro de t\u00e9l\u00e9phone
+phoneNumberVerified=Num\u00e9ro de t\u00e9l\u00e9phone v\u00e9rifi\u00e9
+gender=Sexe
+birthday=Date de naissance
+zoneinfo=Fuseau horaire
 gssDelegationCredential=Accr\u00e9ditation de d\u00e9l\u00e9gation GSS
+logoutOtherSessions=Se d\u00e9connecter des autres appereils
```
[the remark this answers is not in the document]
Comment (Crow-EH) [reply themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties]: done

File: themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties [hunk 4]
```diff
@@ -281,3 +365,26 @@ noCertificate=[Pas de certificat]
...
 internalServerError=Une erreur interne du serveur s''est produite
+
+console-username=Nom d''utilisateur:
+console-password=Mot de passe:
+console-otp=Code \u00e0 usage unique:
+console-new-password=Nouveau mot de passe:
+console-confirm-password=Confirmez les mot de passe:
+console-update-password=La mise \u00e0 jour de votre mot de passe est requise.
+console-verify-email=Vous devez v\u00e9rifier votre adresse de courriel. Nous avons envoy\u00e9 un courriel \u00e0 {0} contenant un code de v\u00e9rification. Veuillez saisir ce code ci-dessous.
+console-email-code=Code Courriel:
```
[the remark this answers is not in the document]
Comment (Crow-EH) [reply themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties]: done

**Review by ssilvert — DISMISSED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**